spec.bs

```diff
@@ -1819,6 +1826,105 @@ Several APIs specific to fenced frames are defined on the {{Fence}} interface.
   </wpt>
 </div>
 
+<div algorithm>
+  The <dfn method for=Fence>disableUntrustedNetwork()</dfn> method steps are:
+
+  1. Let |p| be [=a new promise=].
+
+  1. Let |instance| be [=this=]'s [=relevant global object=]'s [=Window/browsing context=]'s
+     [=browsing context/fenced frame config instance=].
+
+  1. If |instance| is null, then [=resolve=] |p| with {{undefined}} and return |p|.
```

Comment:
Hmm...I actually don't think this will ever be hit. The [fence getter steps](https://wicg.github.io/fenced-frame/#dom-window-fence) will always return a null fence if the browsing context's ff config instance is null, so it would never even be able to access this algorithm. So this could either be an assert, or just no check since we're guaranteed to have a fenced frame config instance by this point. This also applies to the other fence algorithms (since they all have a similar check in place), so those can probably be removed in a follow up.

For the cases you listed:

- The URN iframe case is hit in step 5.
- Iframes nested in fenced frames are allowed to disable network access, so long as they're same origin.
- The cross-origin subframe case is hit in step 4.

Reply:
> so those can probably be removed in a follow up.

Great, could you file a bug and make sure the work that would need to be done to fix that bug matches the implementation too?